### DISCUSSION TITLE
Lone annotations in videos not visible during playback

### DISCUSSION THREAD

**Discussion by agnanas** [biigle/core#973] [Q&A]
I have an issue when reviewing videos and I mark an annotation after pausing the video. I do not usually bother to flag it in multiple frames to have the tracking. As a result I get a single blotch on my timeline showing what I have marked. But when I play the video it does not really visualise the box I have drawn. It could be that because the box lives on a single frame that it is not visualised long enough to actually show up to the naked eye when playing the video or normal or slower speeds. 

But this makes my job very hard when dealing with these "Single frame lifespan" annotations. Because in order to actually see what the box is around I have to individually click on each annotation in the timeline. 

Any way around it ?

**Comment by agnanas** [comment]
I mean annotations like below. They show up on the timeline but when playing the video, unless you look at the annotation timeline in the bottom you'd not even notice the presence of the box
![image](https://github.com/user-attachments/assets/ee740ec5-7e54-4190-ba9e-68bd962193b1)

**Comment by mzur** [comment]
As you correctly suspected, BIIGLE does not bother to show the annotations as they are valid only for a single video frame (often less than 50 ms). So the only way to see the annotations is to jump to the exact video frame.

Maybe Largo could be an alternative to review these annotations? Or do you rather need something like the "Volume Label Review" annotation mode of the image annotation tool?

**Reply by agnanas** [reply]
Largo does not quite solve my problem. 
this is mostly as a review, and I look for two things. 
1. Are all relevant things flagged in the video?
2. Are the right labels attached to whatever has been flagged.?

Now Largo helps with point 2 where I can review label by label and see if there are any mistakes. But it does not solve point 1 for me in seeing if all the fauna have been spotted. For the same reason, I am not sure if the "Volume Label review" option will solve the problem as it will probably just cycle through existing annotations.

I think the problem I need help with or looking for advice on is when I am rewatching an annotated video (where the annotation lives in one frame), if I come across a fauna in the video, it seems that I cannot know if that has been marked or not without looking at the label timeline section and clicking on it to make the box appear.

**Reply by mzur** [reply]
I see your point but I'm not sure if this can be fixed with a technical solution. Video annotations can span multiple frames for the exact reason that you don't miss objects or annotate them twice. If this doesn't work for you, maybe you can annotate the video at fixed intervals or even extract frames as images and then annotate these.
